features-json/css-grid.json

```diff
@@ -50,7 +50,7 @@
       "13":"a x #2",
       "14":"a x #2",
       "15":"a x #2",
-      "16":"a x #2"
+      "16":"p d #5"
```

Comment:
`p` refers to "polyfill" so I don't think it's what you want. Perhaps in this case to avoid confusion you can just change it to `y #5` since I think there's a good chance it won't be behind a flag once Edge 16 is released.